cves/CVE-2018-13304.yml

```diff
@@ -196,8 +200,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    It is stated in the commit message of the fix that the AFLSmart
```

Comment:
Well that's easy! Cool to see a fuzzer used here, haven't seen any credited yet in any of my CVEs or the other's I've reviewed so far.